Homework_1.py

```diff
@@ -0,0 +1,43 @@
+# importing libraries
+
+import pandas as pd
+import plotly.express as px
+from sklearn.preprocessing import StandardScaler
+
+data = pd.read_csv("/home/osboxes/Downloads/iris.csv")
+df1 = pd.DataFrame(data)
+df = pd.DataFrame(data)
+df.pop("class")
+
+n_array = df.to_numpy()
+
+print(n_array)
+
+mean_value = n_array.mean(axis=0)
+min_value = n_array.min(axis=0)
+max_value = n_array.max(axis=0)
+
+print("mean = ", mean_value)
+print("min = ", min_value)
+print("max = ", max_value)
+
+
+Histogram = px.histogram(df, x="petalLength")
```

Comment:
Please add graphs showing the comparison of 
1. Different columns
2. Different Class of data

Reply:
added graphs.